gnucash/import-export/import-pending-matches.hpp

```diff
@@ -39,6 +39,7 @@ struct GNCPendingMatches
         num_manual_matches{0},
         num_auto_matches{0}
     {};
+    bool all_zero () { return (!num_manual_matches && !num_auto_matches); }
```

Comment:
If you're going to add member functions then you should name the member variables according to the Coding Guidelines, i.e. `m_num_manual_matches` and `m_num_auto_matches`.